code/modules/cargo/packs/organic.dm

```diff
@@ -155,25 +155,21 @@
 	desc = "Why visit the kitchen when you can have five random pizzas in a fraction of the time? \
 			Best prices this side of the galaxy! All deliveries are guaranteed to be 99% anomaly-free."
 	cost = CARGO_CRATE_VALUE * 10 // Best prices this side of the galaxy.
-	contains = list(/obj/item/pizzabox/margherita,
-					/obj/item/pizzabox/mushroom,
-					/obj/item/pizzabox/meat,
-					/obj/item/pizzabox/vegetable,
-					/obj/item/pizzabox/pineapple,
-				)
+	contains = list()
 	crate_name = "pizza crate"
+
 	///Whether we've provided an infinite pizza box already this shift or not.
-	var/static/anomalous_box_provided = FALSE
-	///The percentage chance (per pizza) of this supply pack to spawn an anomalous pizza box.
-	var/anna_molly_box_chance = 1
-	///Total tickets in our figurative lottery (per pizza) to decide if we create a bomb box, and if so what type. 1 to 3 create a bomb. The rest do nothing.
-	var/boombox_tickets = 100
+	var/anomalous_box_provided = FALSE
```

Comment:
Why is this not static anymore?

Reply:
it doesnt need to be, the supply console only makes the datum once